pkg/docker/config/config.go

```diff
@@ -97,6 +112,14 @@ func RemoveAuthentication(sys *types.SystemContext, registry string) error {
 			return false, deleteAuthFromCredHelper(ch, registry)
 		}
 
+		// Next try kernel keyring
+		err := deleteAuthFromKernelKeyring(registry)
+		if err == nil {
+			logrus.Debugf("credentials for %s were deleted from the kernel keyring", registry)
```

Comment:
The log here looks different to the one from SetAuthentication. Let's add the user here as well to have consistent logs.

Reply:
I noticed the difference, but the username is not available in this `func RemoveAuthentication`. :thinking: 
or would it be better if `deleteAuthFromKernelKeyring(registry)` returns (username, pass, error)?

Reply:
Good point! Let´s keep it as is.